README.md

```diff
@@ -1,13 +1,18 @@
 <p align=center>
   <img height="80px" src="https://user-images.githubusercontent.com/1536647/77317442-78625700-6d0b-11ea-9822-0fb21e557e87.png"/>
 </p>
-
-<p align=center> A programming language that understands what a web app is. </p>
+<p align=center>
+  <span>A programming language that understands what a web app is.</span>
+</p>
 <br>
+<p align=center>
+  <a href="https://github.com/wasp-lang/wasp/blob/master/LICENSE"><img alt="license" src="https://img.shields.io/github/license/wasp-lang/wasp"></a>
+  <a href="https://github.com/wasp-lang/wasp/search?l=haskell"><img alt="language" src="https://img.shields.io/badge/language-Haskell-purple.svg"></a>
```

Comment:
Ohoo cool we get haskell language badge :D.

Reply:
Hahaha